templates/supplier_app/Supplier/taxpayer-creation.html

```diff
@@ -1,10 +1,81 @@
 {% extends 'base.html' %}
 {% load i18n %}
 
+{% block styles %}
+{% load static %}
+
+<link rel="stylesheet" type='text/css' href="{% static 'css/taxpayer_creation.css' %}">
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.0/jquery.min.js" integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ=" crossorigin="anonymous"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-easing/1.4.1/jquery.easing.min.js" integrity="sha256-H3cjtrm/ztDeuhCN9I4yh4iN2Ybx/y1RM7rMmAesA0k=" crossorigin="anonymous"></script>
+
+<style>
+    .check>label>input{
+    background-color: red;
+    
+}
```

Comment:
I think this can be in `css/taxpayer_creation.css`, is there any reason to put it here?